libs/core/src/components/pds-icon/docs/pds-icon.mdx

```diff
@@ -11,11 +11,11 @@ categories: Actions and Features. In order to ensure that the icons are adding t
 and not creating confusion, we adhere to two principles.
 
 <DocCanvas client:only mdxSource={{
-  react: `<PdsIcon name="upload"></PdsIcon>`,
-  webComponent: `<pds-icon name="upload"></pds-icon>`
+  react: `<PdsIcon name="youtube"></PdsIcon>`,
```

Comment:
I’m on mobile so I can’t check this visually so I’m curious if we want to avoid a branded icon example here? If so, maybe instead of YouTube maybe using the Kajabi icon or some other generic one?

Reply:
Took your advice and changed to `thumb-up-filled`